CONTRIBUTING.md

```diff
@@ -47,4 +61,15 @@ Describe the change you've made in the PR description, put an issue number if ex
 Before submitting a PR, format the changed .cpp/.c/.h files with `clang-format` (included into LLVM distribution). For example:
 `clang-format -i src/ispc.cpp`.
 
-Squash git history to meaningful commits. One commit is responsible for one change.
+Squash git history to meaningful commits. One commit is responsible for one change for easier review.
+
+
+### Contribute a feature or maintain a target
+
+We are looking for individual, academic and institutional contributors who are interested in more significant contributions than a single fix as well. The possible list of topic include:
+- Fine-tuning existing targets, x86 and non-x86. Contributions to ARM targets are especially appreciated, as the core team doesn't have enough bandwidth to fully cover this.
+- Adding and maintaining a new hardware targets - Webassembly (WASM), RISC-V, ARM v9, etc.
+- Adding and/or maintaining a new supported OS/platform - PS4/PS5 (hey Sony, I'm calling for you!), Windows on ARM, FreeBSD, etc.
```

Comment:
Seems to me that the word "supported " is not needed here: "Adding and/or maintaining a new supported OS/platform"